.gitattributes

```diff
@@ -1,6 +1,13 @@
+# Default behaviour, for if core.autocrlf isn't set
+* text=auto
+
+# Binary files
 boot/ocamlc binary
 boot/ocamllex binary
 boot/ocamldep binary
+*.gif
+*.png
+*.tfm
```

Comment:
No need to change, I was just surprised. (The import of `manual/` is a recent addition and I had no idea there were pictures in them.) Thanks for the clarification!